app/models/user.rb

```diff
@@ -34,6 +34,7 @@
 #  image_content_type  :string
 #  auth_provider       :string
 #  home_tile           :integer
+#  tou_agreed          :datetime
```

Comment:
What does this map in PostgreSQL? We should be using `timestamp with time zone` for a purpose like this.

Reply:
All out servers run on UTC anyway so it won't make any real difference.

Reply:
As @tomhughes says and we use UTC for timestamps in UTC in other places of the code most particularly for CT acceptance so I would find it a bit confusing to treat this differently.

Reply:
We should stick with `timestamp (without time zone)` for consistency in this PR. But as @pnorman indicates, we should migrate all timestamp fields to `timestamp with time zone` as a separate task.

Reply:
@gravitystorm : please keep in mind that changing to "timestamp with time zone" possibly also has some impact on cgimap, it's not a rails only thing.

Reply:
I don't think it should matter given that we always use UTC anyway. I equally don't see why we would want to change either but that's not really a matter for this PR.

Reply:
I note that the ongoing discussion currently makes it impossible to finalize this PR, it would be really nice if this could be done in this year.